packages/myst-execute/tests/execute.yml

```diff
@@ -171,9 +171,9 @@ cases:
                   text: |
                     abc
         - type: block
+          kind: notebook-code
```

Comment:
Should we add additional tests with more nesting (as mentioned in #1134 - I think this would be the right place for them...?)

Reply:
added a nested block with non-executable code!